boost.yaml

```diff
@@ -0,0 +1,12 @@
+name: boost
+buildsystem: simple
+sources:
+  - type: archive
+    url: https://dl.bintray.com/boostorg/release/1.68.0/source/boost_1_68_0.tar.bz2
+    sha256: 7f6130bc3cf65f56a618888ce9d5ea704fa10b462be126ad053e80e553d6d8b7
+build-commands:
+  - ./bootstrap.sh --prefix="${FLATPAK_DEST}" --with-libraries=filesystem,locale,system,regex,thread,test;
+  - ./b2 headers;
+  - ./b2 install;
+post-install:
+  - install -p -D -m 0644 "LICENSE_1_0.txt" -t "${FLATPAK_DEST}/share/licenses/boost/";
```

Comment:
Not needed.

Reply:
> Not needed.

You may not care about license compliance, but I and others do.
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L42
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L56
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L66
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L80
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L92
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L106
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L118
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L130
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L155
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L205
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L216
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L254
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L270
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L286
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L304
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L320
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L336
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L344
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L367
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L396
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L416
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L426
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L439

See also:
https://github.com/flathub/flathub/pull/72#commits-pushed-f8778fe
```
While the legal questions of Flathub are being worked out this takes
some basic steps to ensure license compliance.

Most of the time, license compliance is as simple as including the
license and/or copyright file with distribution of the application.
In traditional linux packaging it is common to have a post install
step that does this. Fortunately flatpak-builder can do this too.
```

This is de facto required by law and I don't care whether these files are exported by flatpak or not - they must be included in the package.